docs/dev_quickstart.md

```diff
@@ -97,19 +97,19 @@ Requires only [docker compose](https://docs.docker.com/compose/) and [yarn 1.x](
   ```
   Typically the app is then served at http://localhost:9090 (look at the console to see actual port number)
 
-## Tips
+## FAQ
```

Comment:
Personally prefer starting the title with the most important word, allows for quickly scanning the docs.

Reply:
what would you recommend here?

Reply:
Tips of FAQ ? , both are fine IMO , more a general note on starting labels/headers with a significant word ( instead of , How to ... , What ... , ).  ( nice tips at : https://material.io/design/communication/writing.html#principles , section "Write simply and directly"